airgun/widgets.py

```diff
@@ -2138,7 +2180,7 @@ def fill(self, value):
         self.clear()
         super().fill(value)
         self.browser.plugin.ensure_page_safe()
-        self.browser.send_keys_to_focused_element('\t')
```

Comment:
This is undoing the fix applied in #718, but the relevant PR in foreman isn't in yet, why is this being backed out?

Reply:
https://github.com/SatelliteQE/airgun/pull/718/files#r904462035

Reply:
adding it back in.